### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by Hades-HY-LI** [USGS-R/smwrStats#16]
I have some troubles with installation the package.
I followed the steps in the Package Installation instruction, but when I tried `install_github("USGS-R/smwrStats")`. It shows:

Downloading GitHub repo USGS-R/smwrStats@HEAD
Skipping 2 packages not available: smwrGraphs, smwrBase
Running `R CMD build`...
* checking for file ‘/private/var/folders/6n/7h46gnfn4c55vhvsmz5zl_gw0000gn/T/Rtmp0ypdfv/remotes1bac36cb8384/USGS-R-smwrStats-2e02eb7/DESCRIPTION’ ... OK
* preparing ‘smwrStats’:
* checking DESCRIPTION meta-information ... OK
* cleaning src
* checking for LF line-endings in source and make files and shell scripts
* checking for empty or unneeded directories
* building ‘smwrStats_0.7.6.tar.gz’
* installing *source* package ‘smwrStats’ ...
** using staged installation
** libs
/opt/R/arm64/bin/gfortran -mtune=native -fno-optimize-sibling-calls  -fPIC  -Wall -g -O2  -c d_logam.f -o d_logam.o
make: /opt/R/arm64/bin/gfortran: No such file or directory
make: *** [d_logam.o] Error 1
ERROR: compilation failed for package ‘smwrStats’
* removing ‘/Library/Frameworks/R.framework/Versions/4.1-arm64/Resources/library/smwrStats’
Warning message:
In i.p(...) :
  installation of package ‘/var/folders/6n/7h46gnfn4c55vhvsmz5zl_gw0000gn/T//Rtmp0ypdfv/file1bac5037c3d/smwrStats_0.7.6.tar.gz’ had non-zero exit status

Could you help me with this? Thank you.

**Comment by ldecicco-USGS** [comment]
I think that means you don't have a fortran complier that R knows about. The first thing I'd try would be to install "Rtools":
https://cran.r-project.org/bin/windows/Rtools/rtools40.html
Note...this is an external program to install OUTSIDE of R. It's not an R package.

Once you've installed that, make sure to restart R. Let me know if that doesn't work and we can try something else.

**Comment by sophia-the-dev** [comment]
I am also having the same issue, on mac by the way. Isn't Rtools for windows? I don't think there's a mac alternative for that program. Any other suggestions? 

Thank you!

**Comment by ldecicco-USGS** [comment]
Yeah, Rtools is Windows specific. I have to defer to Mac users on how to deal with Mac installation. @jread-usgs @wdwatkins  or @aappling-usgs or others... can any of you install `remotes::install_github("USGS-R/swmrStats")` and if so, any tips on how to deal with those errors described above?

**Comment by jordansread** [comment]
lol, Laura - you swimmer'd it (swmr vs smwr). I do the same

But no, I can't install using `remotes`
```r
remotes::install_github("USGS-R/smwrStats")
```
![image](https://user-images.githubusercontent.com/2349007/150578978-2ee385df-71c0-4838-9b9c-77fc746a85c6.png)

**Comment by wdwatkins** [comment]
Same error for me.  My best guess would be that the error is related to the warnings immediately above about not being able to see `gfortran` libraries in the paths specified.  There might be ways to specify that location at build time, could involve digging into the makefile to see what the variables are it looks at.

I have vague memories of doing something similar for rloadest years ago.

**Comment by Hades-HY-LI** [comment]
> I think that means you don't have a fortran complier that R knows about. The first thing I'd try would be to install "Rtools": https://cran.r-project.org/bin/windows/Rtools/rtools40.html Note...this is an external program to install OUTSIDE of R. It's not an R package.
> 
> Once you've installed that, make sure to restart R. Let me know if that doesn't work and we can try something else.

**Comment by Hades-HY-LI** [comment]
> Yeah, Rtools is Windows specific. I have to defer to Mac users on how to deal with Mac installation. @jread-usgs @wdwatkins or @aappling-usgs or others... can any of you install `remotes::install_github("USGS-R/swmrStats")` and if so, any tips on how to deal with those errors described above?

My problem is on Mac too, lol. I have tried `remotes::install_github("USGS-R/smwrStats")` before, but it didn't work. I think I have to wait for updation for that. Thanks for your response.

**Comment by ldecicco-USGS** [comment]
Ah, OK. I'm Windows and it does install for me (with Rtools...not sure if that's needed or not). I'd say we should re-open this comment so that other Mac users know there's a problem, and maybe someone can offer a solution.
